cmd/podman/spec.go

```diff
@@ -241,10 +246,16 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 	}
 
 	// Devices
-	for _, device := range config.Devices {
-		if err := addDevice(&g, device); err != nil {
+	if config.Privileged {
```

Comment:
Can you add a comment on this saying there's no point in adding user-provided devices if we're adding all devices?

Reply:
sure